### DISCUSSION TITLE
How to make a proxy server with microsoft remote desktop

### DISCUSSION THREAD

**Discussion by seunggyu0572** [FreeRDP/FreeRDP#7990] [Q&A]
I made a proxy server with freerdp module in java.
but I had a problem with NLA Protocol.
How could I resolve this problem? 
Please reply to me!!

**Comment by akallabeth** [comment]
I´m sorry, but what?

Please try to explain what you try to do and where your problems are in a (more verbose) way that someone not you can retrace the setup and provide some help
